### ISSUE TITLE
Ruby setup

### ISSUE THREAD

**Issue by ManWithBear** [futurice/ios-good-practices#64]
Right now according to your guide gems should be installed by `sudo`, which is [bad](https://github.com/calabash/calabash-ios/wiki/Best-Practice%3A--Never-install-gems-with-sudo).
Suggest to add note about [rvm](https://rvm.io) setup and link to `sudo gem` problem explanation.

**Comment by Sebastian-Hojas** [comment]
I fully agree with that change. In case you are interested, we'd be happy to receive a PR on it.

**Comment by ManWithBear** [comment]
Good point. I will try to find time for it in close future.

**Comment by PH9** [comment]
I prefer `rbenv`, so what should it be? between `rvm` or `rbenv`.

Or we show both cases but not prefer `sudo`?
